sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -525,8 +525,20 @@ object SQLConf {
       .booleanConf
       .createWithDefault(true)
 
+  val COALESCE_PARTITIONS_PARALLELISM_FIRST =
+    buildConf("spark.sql.adaptive.coalescePartitions.parallelismFirst")
```

Comment:
better to update the doc of
ADVISORY_PARTITION_SIZE_IN_BYTES

Reply:
`ADVISORY_PARTITION_SIZE_IN_BYTES` doesn't mention `COALESCE_PARTITIONS_MIN_PARTITION_NUM` either. I think it's fine to only refer to "big" confs in the "small" confs, not from the other side.